app/Repositories/LocationRepository.php

```diff
@@ -173,9 +174,9 @@ public function getLocationDetail($filter)
             ->where('date', $filter['date']);
         if (array_key_exists('shift', $filter) && $filter['shift'] != 0) {
             $listUserId = $listUserId->whereIn(
-                'shift', 
+                'shift',
                 [
-                    $filter['shift'], 
+                    $filter['shift'],
                     config('site.shift.all')
                 ]
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Comma required after last value in array declaration